packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -30,25 +31,42 @@ const DEFAULT_CLI_CONFIG_FILENAME = new URL('client.config.json', DEFAULT_CONFIG
 const LEGACY_CLI_CONFIG_FILENAME = new URL('config.json', DEFAULT_CONFIG_PATH) // todo(1615): Remove this backwards compatibility support
 const DEFAULT_INDEXING_DB_FILENAME = new URL('./indexing.sqlite', DEFAULT_CONFIG_PATH)
 
-const DEFAULT_DAEMON_CONFIG = DaemonConfig.fromObject({
-  anchor: {},
-  'http-api': { 'cors-allowed-origins': [new RegExp('.*')], 'admin-dids': [] },
-  ipfs: { mode: IpfsMode.BUNDLED },
-  logger: { 'log-level': LogLevel.important, 'log-to-files': false },
-  metrics: {
-    'metrics-exporter-enabled': false,
-  },
-  network: { name: Networks.TESTNET_CLAY },
-  node: {},
-  'state-store': {
-    mode: StateStoreMode.FS,
-    'local-directory': DEFAULT_STATE_STORE_DIRECTORY.pathname,
-  },
-  indexing: {
-    db: `sqlite://${DEFAULT_INDEXING_DB_FILENAME.pathname}`,
-    'allow-queries-before-historical-sync': true,
-  },
-})
+/**
+ * Generates a valid Daemon config.
+ *
+ * Most values are set to hardcoded defaults.
+ * The `node.did-seed` is randomly generated.
+ * @returns Daemon config with default values
+ */
+const generateDefaultDaemonConfig = () => {
+  const didSeed = generateSeedUrl()
+  const did = makeNodeDID(parseSeedUrl(didSeed)).id
+
+  return DaemonConfig.fromObject({
+    anchor: {
+      auth: AnchorServiceAuthMethods.DID
+    },
+    'http-api': { 'cors-allowed-origins': [new RegExp('.*')], 'admin-dids': [] },
+    ipfs: { mode: IpfsMode.BUNDLED },
+    logger: { 'log-level': LogLevel.important, 'log-to-files': false },
+    metrics: {
+      'metrics-exporter-enabled': false,
+    },
+    network: { name: Networks.TESTNET_CLAY },
+    node: {
+      'did': did,
+      'did-seed': didSeed
```

Comment:
```suggestion
      'did-private-key': didSeed
```

Using private key instead of seed is more precise here.

Reply:
? it is the seed that generates the private key iiuc
https://github.com/ceramicnetwork/js-did/blob/27a124597af9f61147be48ecca907978579d60ae/packages/key-did-provider-ed25519/src/index.ts#L93-L94

Reply:
Good point. I thought that the seed and the private key was equivalent in this case, but it seems like their not (seed is 32 bytes, private key is 64 bytes). 

Two options:
- Maybe call it `private-seed`, or keep `seed` but don't use the multicode varint format as suggested below (that requires the actual private key to be used
- Use [`extractPublicKeyFromSecretKey`](https://www.stablelib.com/modules/_stablelib_ed25519.html#extractPublicKeyFromSecretKey) instead and use the suggested private key encoding as suggested below.

Reply:
do you think there is any advantage between one or the other in terms of storing the seed vs storing the private key?

Reply:
It's mainly easier to understand what a private key is than what a seed is. I don't feel strongly though.